rock_paper_scissors/game.py

```diff
@@ -0,0 +1,75 @@
+from rock_paper_scissors.valid_choices import VALID_CHOICES
+
+
+class Game(object):
+
```

Comment:
Similar to my above comment, this blank line should be removed.

The exception to this rule is when a class-level docstring is present, [per](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) PEP 257:
> Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.

Rule in a nutshell: 
- Don't use blank lines to separate definitions from functionality
- Don't use blank lines to separate docstrings from functionality
  - _Unless_ it's a class-level docstring